sunpy/visualization/animator/line.py

```diff
@@ -156,4 +156,5 @@ def update_plot(self, val, line, slider):
                 line.set_xdata(self.xdata[tuple(item)])
             slider.cval = val
         # Update slider label to reflect real world values in axis_ranges.
-        super().update_plot(val, slider)
+        im = None
+        super().update_plot(val, im, slider)
```

Comment:
```suggestion
        super().update_plot(val, None, slider)
```

Reply:
Can you check if this happens in other places?

Reply:
Okay, this looks clean. Let me try.